### PR TITLE
[DOCS] Disambiguate logs and data in path settings docs

### LINKED ISSUE #70615: Documentation doesn't describe the difference between logs and data configuring new installations.
Where: current (7.11) documentation.

Who: new users installing elasticsearch

What: how to make decisions on where to put logs and data without being able to find a description of how they are different.

There is a question on the [discuss forum](https://discuss.elastic.co/t/a-post-in-question-on-what-elasticsearch-considers-data-vs-logs-requires-staff-attention/267928) asking this question.

The person answering *does* answer the question:
```
Logs that are ingested into Elasticsearch end up in Indices in Elasticsearch that data is stored in 
path.data example /var/data/elasticsearch

The Logs that Elasticsearch generates in the course of operating are stored in path.logs example 
/var/log/elasticsearch
```

But the [quick docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/important-settings.html#path-settings) they point to *before* that answer doesn't contain that differentiation. 

It is the page where I was when I asked myself the same question. The differentiation should be either on the `important-settings` page or clearly linked from that page.

### PR DESCRIPTION
**Pull request by jrodewig**
The current [Path settings](https://www.elastic.co/guide/en/elasticsearch/reference/7.11/important-settings.html#path-settings) section of [Important Elasticsearch configuration](https://www.elastic.co/guide/en/elasticsearch/reference/7.11/important-settings.html) doesn't distinguish between logs you ingest and ES's own application logs. This could be confusing for users using ES to ingest logs.

This adds a couple of sentences to clarify.

Closes #70615

### CODE REVIEW

File: docs/reference/setup/important-settings/path-settings.asciidoc [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
+directory. {es} writes its own application logs, which contain information about
+cluster health and operations, to a `logs` directory.
+
 For <<targz,macOS `.tar.gz`>>, <<targz,Linux `.tar.gz`>>, and
-<<zip-windows,Windows `.zip`>> installations, {es} writes data and logs to the
-respective `data` and `logs` subdirectories of `$ES_HOME` by default.
-However, files in `$ES_HOME` risk deletion during an upgrade.
+<<zip-windows,Windows `.zip`>> installations, the `data` and `logs` are
+subdirectories of `$ES_HOME` by default. However, files in `$ES_HOME` risk
+deletion during an upgrade.
```
Comment (lockewritesdocs) [inline docs/reference/setup/important-settings/path-settings.asciidoc:12]: Suggest a slight clarification in this sentence, which could be either of these options:
* >the `data` and `logs` directories are subdirectories
* >`data` and `logs` are subdirectories

**Review by lockewritesdocs — APPROVED** [review]
Suggested a minor change, but LGTM otherwise. Thanks @jrodewig!

File: docs/reference/setup/important-settings/path-settings.asciidoc [hunk 2]
```diff
@@ -2,10 +2,14 @@
...
 
+{es} writes the data you index to indices and data streams to a `data`
+directory. {es} writes its own application logs, which contain information about
+cluster health and operations, to a `logs` directory.
+
 For <<targz,macOS `.tar.gz`>>, <<targz,Linux `.tar.gz`>>, and
-<<zip-windows,Windows `.zip`>> installations, {es} writes data and logs to the
-respective `data` and `logs` subdirectories of `$ES_HOME` by default.
-However, files in `$ES_HOME` risk deletion during an upgrade.
+<<zip-windows,Windows `.zip`>> installations, the `data` and `logs` are
```
Comment (jrodewig) [inline docs/reference/setup/important-settings/path-settings.asciidoc:10]:
```suggestion
<<zip-windows,Windows `.zip`>> installations, `data` and `logs` are
```

File: docs/reference/setup/important-settings/path-settings.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/setup/important-settings/path-settings.asciidoc]: Thanks for catching this @lockewritesdocs.

**Pull request merged**